src/Solutions/database.php

```diff
@@ -466,8 +467,11 @@ protected function update($param, $record, $idDoc = null)
             $this->message = 'There is no error but the query hasn\'t modified any record.';
         }
         // Several modifications
-        if ($q->rowCount() > 1) {
-            throw new \Exception('Update query has modified several records. It shoudl never happens. Please check that your id in your database is unique. Query : '.$sql);
+        if (
+				$this->allowSeveralUpdate == false
```

Comment:
**nitpick (non-blocking)**:  Shouldn't this be a === instead of == ?

Reply:
Hi @EstelleMyddleware , you are right. Could you change it ? Thanks.